openff/utilities/provenance.py

```diff
@@ -10,21 +9,28 @@ def _get_conda_list_package_versions() -> Dict[str, str]:
     from openff.utilities.exceptions import CondaExecutableNotFoundError
     from openff.utilities.utilities import has_executable
 
-    if has_executable("mamba"):
+    if has_executable("micromamba"):
+        conda_executable = "micromamba"
+    elif has_executable("mamba"):
         conda_executable = "mamba"
     elif has_executable("conda"):
         conda_executable = "conda"
-    elif has_executable("micromamba"):
-        conda_executable = "micromamba"
     else:
         raise CondaExecutableNotFoundError()
 
-    output = subprocess.check_output([conda_executable, "list"]).decode().split("\n")
+    output = list(
+        filter(
+            lambda x: len(x) > 0,
+            subprocess.check_output([conda_executable, "list"]).decode().split("\n"),
+        )
+    )
```

Comment:
This trick is a slightly different way to handle the difference in ` list` behavior; this just removes the empty line that is present in `micromamba list`